CLI/package.json

```diff
@@ -13,13 +13,13 @@
   "keywords": [
     "citation"
   ],
-  "author": "Eric Lee",
+  "contributors": ["Eric Lee <nokeeo@me.com>", "Robert Lowe <rdl8212@rit.edu>", "Colin O'Neill <cmo6720@rit.edu>", "Samuel Mosher <sam1360@rit.edu>", "mozillafoundation"],
```

Comment:
Could you use <colin@faokryn.com> for my email?

If this is time-sensitive :+1: and I'll update my email later.